src/components/Popover/src/Popover.vue

```diff
@@ -0,0 +1,237 @@
+<script>
+import { throwError } from '@square/maker/utils/debug';
+import assert from '@square/maker/utils/assert';
+import { PopoverConfigKey, PopoverAPIKey } from './keys';
+
+const MAX_ACTION_VNODE = 1;
+
+const getMinWidth = (minWidth, tetherMinWidth, reference) => {
+	if (!tetherMinWidth) {
+		return minWidth;
+	}
+
+	return minWidth || reference.offsetWidth;
+};
+
+const createPopperConfig = ({
+	placement,
+	offset,
+	tetherMinWidth,
+	minWidth,
+}) => ({
+	placement,
+	modifiers: [
+		{
+			name: 'offset',
+			options: { offset },
+		},
+		{
+			name: 'preventOverflow',
+			options: {
+				padding: 24,
+				altBoundary: true,
+			},
+		},
+		{
+			name: 'minWidth',
+			enabled: tetherMinWidth || !!minWidth,
+			phase: 'beforeWrite',
+			requires: ['computeStyles'],
+			fn({ state: { styles, rects } }) {
+				const popoverMinWidth = getMinWidth(minWidth, tetherMinWidth, rects.reference);
+				styles.popper.minWidth = `${popoverMinWidth}px`;
+			},
+			effect({ state: { elements } }) {
+				const popoverMinWidth = getMinWidth(minWidth, tetherMinWidth, elements.reference);
+				elements.popper.style.minWidth = `${popoverMinWidth}px`;
+			},
+		},
+	],
+});
+
+export default {
+	name: 'Popover',
+
+	inject: {
+		popoverAPI: {
```

Comment:
```suggestion
		popoverApi: {
```

minor, but if u can, i would change the casing to match existing `dialogApi`, `modalApi`, etc.